content/docs/2.17/authentication-providers/bound-service-account-token.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Bound service account token"
++++
+
+You can pull a service account token into the trigger by defining the `serviceAccountName` of the Kubernetes ServiceAccount and token `expiry` duration.
+
+```yaml
+boundServiceAccountToken:                        # Optional.
+  - parameter: connectionString                  # Required - Defined by the scale trigger
+    serviceAccountName: my-keda-service-account  # Required.
+    expiry: 1h                                   # Required.
```

Comment:
This parameter isn't supported in the code, I think that it could be interesting to support this override as optional, keeping the current approach of a global value which can be overridden via an optional field here. WDYT?

Reply:
This was written before we went through the review changes in https://github.com/kedacore/keda/pull/6272, which still had the configurable expiry per trigger. I should just remove the `expiry` here. WDYT about leaving the optional expiry feature as an enhancement in a future PR? 

> As default, KEDA doesn't have permission in the RBAC to create these tokens (and not granting it as default is worth IMHO), so maybe we could add an example of the required extra roles that needs to be added to KEDA operator service account (or event better, and example of all the required RBAC manifests)
And as a response to this, yes I believe that would be a great addition to add the required example rbacs to make this work. I will add that in the next commits, thanks!

Reply:
Changed in https://github.com/kedacore/keda-docs/pull/1507/commits/e0133f0479578af903c601709ef21596f4cd3c81.